src/factories/series/Line.ts

```diff
@@ -21,13 +21,53 @@ module n3Charts.Factory.Series {
           .attr('class', 'line-data');
     }
 
-    update(datasets, options) {
-      // Here we will update our line charts
+    _getDrawers(series: Utils.OptionsSeries[]): any {
+      var xScale = <Factory.Axis>this.factoryMgr.get('x-axis');
+      var yScale = <Factory.Axis>this.factoryMgr.get('y-axis');
+
+      var drawers = {};
+
+      series.forEach((s) => {
+        drawers[s.id] = d3.svg.line()
+          .x((d) => xScale.scale(d.x))
+          .y((d) => yScale.scale(d.y));
+      });
+
+      return drawers;
+    }
+
+    update(datasets: Utils.Datasets, options: Utils.Options) {
+      var lines = options.getSeriesForType(Utils.Options.SERIES_TYPES.LINE);
+      var sets: {} = datasets.getValuesForSeries(lines, options);
+      var drawers: {} = this._getDrawers(lines);
+
+      var svgs = this.svg.selectAll('.line-series')
+        .data(
+          lines.map((s) => { return { series: s, set: sets[s.id] }; }),
+          (d) => { return d.series.id; }
+        );
+
+      svgs.enter()
+        .append('g')
+        .attr({
+          class: (d: Utils.OptionsSeries) => 'line-series ' + d.key
+        })
+        .append('path');
+
+      svgs.select('path')
+        .style({
+          'fill': 'none',
+          'stroke': (s) => s.series.color
+        })
+        .transition()
```

Comment:
Could we maybe initialize and use a default Animation object somewhere. Because now it uses default `ease` and `duration` from D3js (lienar, 250ms) - but what if we want to change this. Just a thought